### ISSUE TITLE
Docker swarm creating more worker replicas than allocated on AWS.  

### ISSUE THREAD

**Issue by anyaelena** [NREL/OpenStudio-server#457]
Not yet clear on exact conditions to reproduce, but we have a current example that used 2.7.1-largescale0 ami.  Web node becomes unavailable. It is pending restart because no resources are available, presumably because there are too many worker replicas.

![image001](https://user-images.githubusercontent.com/187915/51624538-47647780-1ef8-11e9-92e9-618404eda0d8.png)

**Comment by brianlball** [comment]
Thats nuts.  Is that related to the possible crash or non-startup of the Web container?

**Comment by anyaelena** [comment]
yes, definitely.  I think this is at the root of a few reported bugs that we've been unable to reliably reproduce.  This is my top priority right now.
